.github/workflows/acceptance-tests-runner.yml

```diff
@@ -151,6 +158,9 @@ env:
   MONGODB_ATLAS_GOV_PRIVATE_KEY: ${{ secrets.mongodb_atlas_gov_private_key }}
   MONGODB_ATLAS_GOV_BASE_URL:  ${{ inputs.mongodb_atlas_gov_base_url }}
   MONGODB_ATLAS_GOV_ORG_ID:  ${{ inputs.mongodb_atlas_gov_org_id }}
+  MONGODB_ATLAS_RP_PUBLIC_KEY: ${{ secrets.mongodb_atlas_rp_public_key }}
```

Comment:
knit: if these env vars are only used in one resource tests, better to move it into its job

Reply:
changed in 77b1c43c3243c8a3e98d4cff79a6af19d7744139